init/clean.js

```diff
@@ -1,31 +1,27 @@
-const { readFile, writeFile } = require('fs')
-const { promisify } = require('util')
+import { readFile, writeFile } from 'fs/promises'
+import { fileURLToPath } from 'url'
 
-const del = require('del')
-const omit = require('omit.js').default
-const filterObj = require('filter-obj')
+import del from 'del'
+import omit from 'omit.js'
+import filterObj from 'filter-obj'
 
-const PACKAGE_ROOT = `${__dirname}/..`
+const PACKAGE_ROOT = fileURLToPath(new URL('..', import.meta.url))
```

Comment:
where is the exact difference between:

`const PACKAGE_ROOT = new URL('..', import.meta.url).pathname`


and using `fileURLToPath`? 🤔

Reply:
`pathname` would retrieve a URL path, which is different from a filesystem path. 

For example, on Windows, URL paths have forward slashes and add a slash in front of the drive letter:
  - `new URL(import.meta.url).pathname`: `/C:/Users/me/netlify/` (incorrect for filesystem usage)
  - `fileURLToPath(new URL(import.meta.url))`: `C:\Users\me\netlify`

Also, URL encoding is different on all OS:
  - `new URL(import.meta.url).pathname`: `/file/to/path/%C3%A9t%C3%A9`
  - `fileURLToPath(new URL(import.meta.url))`: `/file/to/path/été`

Reply:
THX Today I learned :)